### DISCUSSION TITLE
考虑开一个给单测提供帮助的包吗？

### DISCUSSION THREAD

**Discussion by mortalYoung** [ant-design/ant-design#41970] [Ideas]
# 痛点
社区很多组件基于 antd 去封装自己的组件，然后很多组件自己写单测的时候，调用 fireEvent 的时候需要去找源码翻，不同的组件需要翻到 rc 组件库里去。非常麻烦。比如：基于 Select 封装之后，想要测试 Options 里是否有指定的项，或者 onChange 之后是否调用某函数等等。需要去翻 rc-select 里面的单测。

# Resolution
考虑把各个 rc 组件库里的 selectOpen inputChange 之类的都整理一下做一个开源出来造福社区如何？

**Comment by afc163** [comment]
好想法，其实 antd 自己也需要，当下的测试用例有一些封装方法。

楼主来开个新坑？或者先写个 RFC 把 API 梳理出来？

**Reply by mortalYoung** [reply]
我来试试看

**Comment by mortalYoung** [comment]
refer: #42032
